maindata.py

```diff
@@ -4,6 +4,8 @@
 from csvoutput import CsvOutput
 from wikipediafunction import WikipediaFunction
 from youtubefunction import YoutubeFunction
+import web_scraper as webScraper

```

Comment:
I believe this maindata.py is an older version than what's currently in the main repo and therefore is causing a merge conflict. You might want to sync up with the most recent files in the main branch and resubmit your maindata.py